doc/news/changes/incompatibilities/20200928Bangerth

```diff
@@ -0,0 +1,5 @@
+Deprecated: The QTrapez class, poorly named because the proper English
+term is "trapezoidal quadrature rule" has been renamed to QTrapezoid,
+and the old name has been deprecated.
```

Comment:
```suggestion
and the class with the old name has been deprecated.
```

Reply:
That's not actually quite true: The class with the old name simply doesn't exist any more, I just introduced an alias (a "name") that is now deprecated.

But that's a subtlety probably lost on our readers :-) So I went with your suggestion :-)